### ISSUE TITLE
"3.2.7 Hidden Controls" Suggest Clarifying statement about "multiple controls ... in a single location"

### ISSUE THREAD

**Issue by SuzanneTaylor** [w3c/wcag#1402]
The Understanding document states: "When multiple controls are included in a single location, some indication of those controls such as a drop down indicator should be visible without user interaction."

In this context, it could be read two ways:

1. Is this a separate best practice, as in, "While not required to meet SC 3.2.7, for controls that are not needed to move forward in a process, it is still a best practice to provide at least some indication of the presence of any set of controls, such as drop down indicator."
2. Or, is this more to ensure the SC is not interpreted too strictly, as in, 'If a few controls needed to move within a process are grouped together, it is okay if the element that is persistently visible is a container for that set of controls. For example, a "Next Steps" drop down menu might be persistently visible, but require hover, focus or even click to reveal the options, "Start Over," "Spell Check" and "Submit".' 

If the second is the intent, I think it is important to suggest, as in the text suggested above, that this should be a group of the controls needed to move or continue within a process, this way placing the controls needed to move forward in a hidden general menu with a drop down indicator wouldn't be mistaken as a good way to meet this guideline.

**Comment by rachaelbradley** [comment]
The author of issue 1400 suggested changing the text from "When multiple controls are included in a single location, some indication of those controls such as a drop down indicator should be visible without user interaction." to
“If a few controls needed to move within a process are grouped together, it is okay if the element that is persistently visible is a container for that set of controls. “ Would a change such as this address your concern or would additional language discouraging buried controls be needed?

**Comment by SuzanneTaylor** [comment]
Well, I'm not sure. I was just pointing out that the sentence can be read in two ways. I haven't see the research/case descriptions leading to this guideline.

**Comment by alastc** [comment]
HI @SuzanneTaylor,

Thanks for raising this. The [latest version](https://w3c.github.io/wcag/understanding/visible-controls.html) uses a different approach which focuses on the information for that control being available is visible. The text you referred to is not there any more.

I'll close this as the source of the issue is gone, but please do raise any new issues on the new text, it will be going out for review again.
